package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TRIM API wrapper",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/mocha/bin/mocha specs.js",
+    "test:watch": "./node_modules/nodemon/bin/nodemon.js --exec \"npm run test\" specs.js"
```

Comment:
Maybe something like: https://github.com/mediasuitenz/mappy/blob/master/test/key.spec.js

We were using Mocha GWT and arrow functions to clean up the test cases.
I did a follow up with Justin at Test Double and he recommended we drop the text in the Givens, Whens and Thens though as he felt it creates noise so an ideal test looks something like:

```js
describe('setting the maps title', () => {
  Given(() => title = 'My title')
  When(() => key.title = title)
  Then(() => key.title.should.equal(title))
})
```